### DISCUSSION TITLE
accountability: defaultTo vs update

### DISCUSSION THREAD

**Discussion by anlexN** [directus/directus#15471] [Archive]
![image](https://user-images.githubusercontent.com/8686739/189065425-7527c8f2-7aae-43ab-a689-67f3946c3782.png)
can we remove `update` statement?

**Comment by rijkvanzanten** [comment]
If you set a default value to a column, it doesn't necessarily set the current value of that column too. In this case, we need both the defaultTo and update calls.

**Reply by anlexN** [reply]
@rijkvanzanten teacher, I have used Postgresql to default, as it default, not your says situation. what database do you use?
